metadata-ingestion/src/datahub/ingestion/source/dremio/dremio_config.py

```diff
@@ -0,0 +1,184 @@
+        return value
+
+
+class ProfileConfig(GEProfilingConfig):
+    partition_profiling_enabled: bool = Field(
+        default=False,
+        description="Partition profiling disabled for Dremio.",
+    )
+    include_field_median_value: bool = Field(
+        default=False,
+        description="Median causes a number of issues in Dremio.",
+    )
+    query_timeout: int = Field(
+        default=300, description="Time before cancelling Dremio profiling query"
+    )
+
+    row_count: bool = True
+    column_count: bool = True
+    sample_values: bool = True
+
+
+class DremioSourceMapping(EnvConfigMixin, PlatformInstanceConfigMixin, ConfigModel):
+    platform: Optional[str] = Field(
+        default=None,
+        description="Source connection made by Dremio (e.g. S3, Snowflake)",
+    )
+    source_name: Optional[str] = Field(
+        default=None,
+        description="Alias of platform in Dremio connection",
+    )
+
+
+class DremioSourceConfig(
+    DremioConnectionConfig,
+    StatefulIngestionConfigBase,
+    EnvConfigMixin,
+    PlatformInstanceConfigMixin,
+):
+
+    domain: Optional[str] = Field(
+        default=None,
+        description="Domain for all source objects.",
+    )
+
+    source_mappings: Optional[List[DremioSourceMapping]] = Field(
+        default=None,
+        description="Mappings from Dremio sources to DataHub platforms and datasets.",
+    )
+
+    # Entity Filters
+    schema_pattern: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Regex patterns for schemas to filter",
+    )
+
+    dataset_pattern: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Regex patterns for tables and views to filter in ingestion. Specify regex to match the entire table name in dremio.schema.table format. e.g. to match all tables starting with customer in Customer database and public schema, use the regex 'dremio.public.customer.*'",
+    )
+
+    usage: BaseUsageConfig = Field(
+        description="The usage config to use when generating usage statistics",
+        default=BaseUsageConfig(),
+    )
+
+    # Profiling
+    profile_pattern: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Regex patterns for tables to profile",
+    )
+    profiling: ProfileConfig = Field(
+        default=ProfileConfig(),
+        description="Configuration for profiling",
+    )
+
+    # Advanced Configs
+    max_workers: int = Field(
+        default=5 * (os.cpu_count() or 4),
+        description="Number of worker threads to use for parallel processing",
+    )
+
+    include_query_lineage: bool = Field(
+        default=False,
+        description="Whether to include query-based lineage information.",
+    )
+
+    include_table_rename_lineage: bool = Field(
+        default=True,
+        description="Whether to include table rename lineage",
+    )
+
+    include_copy_lineage: bool = Field(
```

Comment:
I believe `include_copy_lineage` and ` include_table_rename_lineage` are not used anywhere ?

Reply:
Yes, removed